syntax = "proto3";

package suggestions;

message SuggestionRequest {
  string book_title = 1;
}

message BookSuggestion {
  string title = 1;
  string author = 2;
}

message SuggestionResponse {
  repeated BookSuggestion suggestions = 1;
}

service SuggestionService {
  rpc GetSuggestions (SuggestionRequest) returns (SuggestionResponse);
}
